srcpkgs/minitube/template

```diff
@@ -1,17 +1,20 @@
 # Template file for 'minitube'
 pkgname=minitube
-version=2.9
+version=3.0
 revision=1
 build_style=qmake
-makedepends="qt5-devel phonon-qt5-devel qt5-script-devel qt5-declarative-devel"
-hostmakedepends="qt5-tools qt5-qmake $makedepends"
+hostmakedepends="qt5-tools qt5-qmake git"
+makedepends="qt5-devel phonon-qt5-devel qt5-script-devel qt5-declarative-devel
+ qt5-plugin-sqlite qt5-x11extras-devel mpv-devel"
 depends="virtual?phonon-qt5-backend desktop-file-utils hicolor-icon-theme"
-short_desc="A native YouTube client in Qt5"
+short_desc="Qt5-based YouTube client"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
-license="GPL-3"
+license="GPL-3-or-later"
 homepage="http://flavio.tordini.org/minitube"
-distfiles="https://github.com/flaviotordini/minitube/archive/$version.tar.gz"
-checksum=0f51fb86d0fdfbcf00af142868ceeacb6c27eb8a1e36d856162e23d63e90eb07
+
+do_fetch() {
+	git clone --recursive https://github.com/flaviotordini/minitube.git ${wrksrc}
```

Comment:
i don't like it, can't the submodules be put in distfiles= ?

Reply:
I agree.  I'm not a fan.  The submodules do not have tarballs available.  I had asked the maintainer to make them via Github so the main package and 3 submodules can be downloaded and checksums verified.

https://github.com/flaviotordini/minitube/issues/133